source/common/network/lc_trie.h

```diff
@@ -42,7 +42,7 @@ class LcTrie {
    *                              factor. It reduces the depth of the trie.
    */
   LcTrie(const std::vector<std::pair<std::string, std::vector<Address::CidrRange>>>& tag_data,
-         double fill_factor = 0.5, uint32_t root_branching_factor = 16);
+         double fill_factor = 0.5, uint32_t root_branching_factor = 0);
```

Comment:
Should this have stayed 16, or should we remove the comment on how 16 is optimal?

Reply:
Earlier @brian-pane made a comment about 16 most likely being optimal for larger data sets and that maybe 4 or 8 would be better. Since we don't have a way to figure out a good default yet(i'm thinking perf tests could help us get that information), I prefer to let the algorithm determine the root branching factor. If consumers have a preference for the values, the root branching factor and fill factor can be exposed in the config. 

I'll update the comment in the trie.